scripts/setup/install_hol4_latest.sh

```diff
@@ -15,15 +15,13 @@ source "${SETUP_DIR}/env_config_gen.sh" "${OPT_DIR_PARAM}"
 
 ##################################################################
 
-
-# make polyml binaries and libraries available
+# use a default polyml version if it is not specified in the environment
 POLY_VERSION="v5.8.1"
```

Comment:
should we change the default here to 5.9 as well?

Reply:
Ah yes, I missed that. I'll add it.